system-design/margo-overview/personas.md

```diff
@@ -1,37 +1,70 @@
 # Personas
 
-**End User**
+Throughout this documentation, the following personas are used to represent interaction with the standard and/or with entities defined by the standard.
 
-Within the industrial automation market interactions are business to business operations. Meaning, end user means the business organization within the transaction. Any user within a company that has access to the Margo compliant application or devices. 
+## Persona Relationships
+![Persona Relationships](./assets/persona-relationships.png)
 
-**Application Developer** 
+## Categories
 
-Application developer persona represents an entity that utilizes the Margo standard to construct their application. By utilizing the Margo standard, the developer can focus on differentiation while inheriting the advantages of the Margo ecosystem.  
+These categories group common personas together under a shared goal or role. These categories will be used in other documentation to provide grouping and clarity where appropriate.
 
-**Application Marketplace Provider**
+### End Users
+- Consumes products provided by the suppliers
+- Evaluates the best-in-class products offered by the suppliers
+- Assembles multiple vendor's products into a productive ecosystem
 
-This Marketplace provider is a transactional software that is provided to End Users to browse available applications and gain access via purchasing the software.  
+### Suppliers
+- Contributes to the Margo specification for their component
+- Builds products aligned to the Margo standard
+- Markets and sells their products to end users
 
-Note:
-Application Marketplaces are out of scope for Project Margo. However, Margo provides guidelines around content required within the Application Package that is utilized by Marketplaces.  
+## Persona Definitions
 
-**Workload Orchestration Vendor** 
+These persona definitions represent individual entities with specified use cases and requirements, aligned with the other personas in their category.
 
-The Workload Orchestration Vendor provides End Users with the infrastructure to manage applications along with Kubernetes Node and Node Cluster enrollment and management.   
+### End Users
 
-**Device Owner**
-The Device Owner within Project Margo is responsible for meeting the requirements outlined within the device requirements section.
+End users consume products and technologies from the suppliers to drive business continuity and efficiency
 
-[Device Requirements](../device-interoperability/device-requirements.md)
+#### OT User
 
-**Device Manufacturer**
+Consumer of functionality provided by the application vendors to run critical and non-critical business functions, or to improve business efficiency
```

Comment:
This definition seems to fuzzy for me in a clear context as we have: industrial automation.

Reply:
What would you suggest as a clearer definition?